content/3_Silver/Prefix_Sums_2.mdx

```diff
@@ -102,11 +102,96 @@ export const problems = {
 
 <FocusProblem problem={problems.maxsum} />
 
+### Solution
+
+<Spoiler title="Solution">
+
+Suppose that we store the prefix sums at each index in the array. Intuitively, the maximum possible $p[i]-p[j]$ where $j\leq i$. Thus, we can iterate each element in the array and store the minimum prefix sum up to the current index. Then, we can find the maximum subarray sum by taking the difference of the current prefix sum and the minimum prefix sum.
```

Comment:
```suggestion
Consider the prefix sum array. The maximum subarray sum is the maximum possible $p[i]-p[j]$ where $j\leq i$. We can iterate over each index $i$ and keep a running minimum $p[j]$ value. This yields the maximum subarray sum for each possible right bound, and the maximum over all these values is our answer.
```

I think this is more clear. Maybe play around w the wording to make it more understandable.